src/HW_models/NRF_RTC.c

```diff
@@ -336,6 +385,11 @@ void nrf_rtc_regw_sideeffect_INTENSET(int i) {
     RTC_regs->INTENSET = RTC_INTEN[i];
     check_not_supported_func(RTC_INTEN[i]);
   }
+
+  if (RTC_regs->INTENSET & RTC_INTENSET_TICK_Msk){
+    bs_trace_raw_time(8, "RTC: Simulations are slower when the tick event is enabled\n");
+    update_next_match_events_tick(i);
```

Comment:
To be sure `Timer_RTC` is accounted for, `nrf_hw_find_next_timer_to_trigger();` needs to be called